### PR TITLE
net-misc/bird: fixing tests on MUSL

### PR DESCRIPTION
**Pull request by alarig**
Closes: https://bugs.gentoo.org/836713

### CODE REVIEW

File: net-misc/bird/bird-2.0.10.ebuild [hunk 1]
```diff
@@ -36,8 +36,15 @@ FILECAPS=(
 	CAP_NET_RAW				usr/sbin/bird
 )
 
+PATCHES=()
+
 src_prepare() {
+	if use elibc_musl; then
```
Comment (thesamesam) [inline net-misc/bird/bird-2.0.10.ebuild:42]: Please apply it unconditionally - it should be safe.

See also https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#conditional-patching.

File: net-misc/bird/files/bird-2.0.9-musl-tests.patch [hunk 2]
```diff
@@ -0,0 +1,40 @@
+From 0f679438f36d8c2a31dfe490007e983b085caef6 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline net-misc/bird/files/bird-2.0.9-musl-tests.patch:1]: Link to the commit / ML post if you can too

File: net-misc/bird/bird-2.0.10.ebuild (hunk 1 above)
Comment (alarig) [reply net-misc/bird/bird-2.0.10.ebuild]: I didn’t know, thanks for the link! I modified the two files.

File: net-misc/bird/files/bird-2.0.9-musl-tests.patch (hunk 2 above)
Comment (alarig) [reply net-misc/bird/files/bird-2.0.9-musl-tests.patch:4]: I’ve added the two links

**Pull request closed without merging**